pom.xml

```diff
@@ -67,7 +67,7 @@
   <properties>
     <revision>2.6.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.308</jenkins.version>
+    <jenkins.version>2.321</jenkins.version>
```

Comment:
Using the newest weekly release makes this and the bindings plugin incompatible with any LTS version again. 
This should be postponed after #248 is merged imo, so the credentials plugin is available to at least one LTS for now again, and 2.319 in a couple of weeks.

Reply:
Also you should switch to `bom-weekly`.